### PR TITLE
Parameter tuning and documentation update

### PR DESCRIPTION
**Pull request by KedoKudo**
This PR introduces the following changes:

- parameter tuning study results
- default parameters are updated based on results above
- update documentation on installation and development environment setup

How to test
------------
1. Clone the repo and run the example notebooks.
2. Run the unit tests.

Known issues
-------------
1. The multi-scale step does not provide expected denoising improvement, and a [story](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=6102) is created to track corresponding work.

> related EWM item: [Story 4899](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=4899)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +12,41 @@ The BM3D ORNL code is designed to be easy to use and easy to integrate into exis
 The BM3D ORNL code is released under an open-source license, and is freely available for download and use.
 
 For more information, check out our [FAQ](docs/FAQ.md).
+
+How to install
+--------------
+
+For **users**, you can install the latest version published on our anaconda channel with:
```
Comment (peterfpeterson) [inline README.md:19]: Add a link to the conda package page

File: README.md [hunk 2]
```diff
@@ -11,3 +12,41 @@ The BM3D ORNL code is designed to be easy to use and easy to integrate into exis
...
+
+```bash
+conda install neutronimaging::bm3dornl
+```
+
+Or use `pip` to install from PyPI
```
Comment (peterfpeterson) [inline README.md:25]: same for pypi

File: README.md [hunk 3]
```diff
@@ -11,3 +12,41 @@ The BM3D ORNL code is designed to be easy to use and easy to integrate into exis
...
+conda env create -f environment.yml
+```
+
+followed by
+
+```bash
+pip install --no-deps -e .
+```
+
+The option `--no-deps` here is critical as `pip` will try to install a second set of dependencies based on information from `pyproject.toml`.
```
Comment (peterfpeterson) [inline README.md:49]: This is an excellent observation/comment. Can you copy it into the python template?

File: src/bm3dornl/bm3d.py [hunk 4]
```diff
@@ -294,47 +309,20 @@ def estimate_noise_free_sinogram(sinogram: np.ndarray) -> np.ndarray:
...
-        "background_threshold": 0.0,
-        "cut_off_distance": (64, 64),
-        "num_patches_per_group": 32,
-        "padding_mode": "circular",
-    },
-    filter_kwargs: dict = {
-        "filter_function": "fft",
-        "shrinkage_factor": 3e-2,
-    },
+    block_matching_kwargs: dict = default_block_matching_kwargs,
```
Comment (peterfpeterson) [inline src/bm3dornl/bm3d.py:324]: Does this need to be a deep copy?

**Review by peterfpeterson — COMMENTED** [review]
I only marked one place with a question about `block_matching_kwargs` and `filter_kwargs` applies to all of the occurences

File: README.md (hunk 2 above)
Comment (KedoKudo) [reply README.md]: link is added

File: README.md (hunk 3 above)
Comment (KedoKudo) [reply README.md:49]: will do

**Review by peterfpeterson — APPROVED** [review]

**Pull request merged**